docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -142,103 +137,6 @@ Note the following:
-CREATE TABLE employees_2020_12 () INHERITS (employees);
-CREATE TABLE employees_2021_01 () INHERITS (employees);
-```
-
-The next step is to add non-overlapping table constraints to the partition tables to define the key values in each partition, as shown in the following example:
-
-```sql
-CHECK ( n = 1 )
-CHECK ( department IN ( 'Sales', 'Marketing'))
-```
-
-The following example shows how to create partitions:
-
-```sql
-CREATE TABLE employees_2019_02 (
-  CHECK ( change_date >= DATE '2019-02-01' AND change_date < DATE '2019-03-01' )
-) INHERITS (employees);
-
-CREATE TABLE measurement_2019_03 (
-  CHECK ( change_date >= DATE '2019-03-01' AND change_date < DATE '2019-04-01' )
-) INHERITS (employees);
-...
-CREATE TABLE measurement_2020_11 (
-  CHECK ( change_date >= DATE '2020-11-01' AND change_date < DATE '2020-12-01' )
-) INHERITS (employees);
-
-CREATE TABLE measurement_2020_12 (
-  CHECK ( change_date >= DATE '2020-12-01' AND change_date < DATE '2020-01-01' )
-) INHERITS (employees);
-
-CREATE TABLE measurement_2021_01 (
-  CHECK ( change_date >= DATE '2021-01-01' AND change_date < DATE '2021-02-01' )
-) INHERITS (employees);
-```
-
-The following example shows how to create an index on the key columns and, optionally, other indexes on partitions:
-
-```sql
-CREATE INDEX employees_2019_02 ON employees_2019_02 (change_date);
-CREATE INDEX employees_2019_03 ON employees_2019_03 (change_date);
-CREATE INDEX employees_2020_11 ON employees_2020_11 (change_date);
-CREATE INDEX employees_2020_12 ON employees_2020_12 (change_date);
-CREATE INDEX employees_2021_01 ON employees_2021_01 (change_date);
-```
-
-The following example shows how to quickly remove old data via dropping the partition:
-
-```sql
-DROP TABLE employees_2019_02;
-```
-
-The following example shows how remove the partition from the partitioned table while retaining access to it as a separate table:
-
-```sql
-ALTER TABLE employees_2019_02 NO INHERIT employees;
-```
-
-The following example shows how to add a new partition to deal with new data by creating an empty partition:
-
-```sql
-CREATE TABLE employees_2021_02
-  CHECK ( change_date >= DATE '2021-02-01' AND change_date < DATE '2021-03-01')
-  TABLESPACE fasttablespace;
-```
-
-If your goal is to have data loaded, verified, and modified before it is added to the partitioned table, you can create a new table outside of the partition structure: 
-
-```sql
-CREATE TABLE employees_2021_02
-  (LIKE employees INCLUDING DEFAULTS INCLUDING CONSTRAINTS);
-ALTER TABLE employees_2021_02 ADD CONSTRAINT 2021_02
-  CHECK ( change_date >= DATE '2021-02-01' AND change_date < DATE '2021-03-01' );
-...
-ALTER TABLE employees_2021_02 INHERIT employees;
-```
-
-Note the following:
-
-- When using  `VACUUM` or `ANALYZE` commands, you have to run them on each partition. 
-- Triggers or rules will be needed To direct rows to the desired partition, you need to use rules or triggers. 
-
 ## Constraint Exclusion
```

Comment:
This section is nowhere as clear as the equivalent in Postgres documentation unfortunately: https://www.postgresql.org/docs/11/ddl-partitioning.html#DDL-PARTITIONING-CONSTRAINT-EXCLUSION

Also weirdly why have we chosen this section (which is more applicable for legacy inheritance) rather than partition pruning? Why have we also not chosen the section on Partitioning Best Practices?

Reply:
Our doc site is not structured like postgres docs. Our Explore section contains overview-type articles only. 
Thanks for your input.